### DISCUSSION TITLE
Hilfe, meine openDTU on battery kann meinen HM-1500 nicht regeln

### DISCUSSION THREAD

**Discussion by Maddi1975** [hoylabs/OpenDTU-OnBattery#1084] [Q&A] [answered]
Moin zusammen,
ich brauche dringend Hilfe. Ich habe ein BKW 4x500Watt an einem Zendure HUB2000 + 2x AB2000. Vom HUB gehe ich per Y-Kabel auf alle 4 Eingänge des HM-1500. Außerdem habe ich einen Shelly Pro 3EM verbaut. Hierdurch habe ich eine Nulleinspeisung über den CT-Mode des Zendure laufen. der läuft im Prinzip ganz ok, bis auf das er etwas träge ist, und der Bypassmode für mich nicht funktioniert. Nun wollte ich eine Nulleinspeisung über die OpenDTU on battery laufen lasen und versuche dies seit Tagen zum laufen zu bekommen.... Leider ohne Erfolg. Ich bin mir fast sicher, die OpenDTU vernünftig konfiguriert zu haben..

![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/174735302/80ed3fa9-68f6-4f08-93b4-8b549ffc2a4f)

Das Problem ist, dass das von der DTU gesetzte Limit des Wechselrichters nicht vom Wechselrichter übernommen wird. Wie gesagt, läuft die Regelung über Zendure und den CT Mode ist alles ok.

Hier sieht man, was ich meine:
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/174735302/c141829a-a930-44b0-8425-bc69e3b0caba)

Hoffe mir kann hierbei jemand helfen.

Beste Grüße,
Mad

**Comment by spcqike** [comment, marked as answer]
Wie hast du das denn alles angeschlossen und wie regelt der zendure den Ausgang? Gibt we die Batterie voll durch oder drosselt der eventuell auch? 

Ich finde es komisch, dass Eingang 1 und 2 jeweils so viel erzeugen, 3 und 4 hingegen so wenig. 

Der HM-1500 hat 4 Eingänge aber nur 2 MPPT, heißt es sind jeweils 2 Eingänge intern parallel. Bei symmetrischem Anschluss und keine drosselung seitens der Quelle, sollten alle Eingänge gleich viel erzeugen. 

Du möchtest 750W AC, d.h. Jeder MPPT steuert bis zu 375W AC, bei 95% Effizienz also ~ 395W DC bei. 

Eingang 1&2 liefern also bereits zu viel, Eingang 3&4 hingegen zu wenig

**Comment by Maddi1975** [comment]
Hey, danke für die fixe Antwort.
Wenn ich auf openDTU Regelung stelle, dann sind im Zendure alle Mode´s abgewählt und die Ausgangsleistung ist auf max.
Am Hub ist ja nur ein Ausgang, diesen splitte ich auf 2 mit den mitgelieferten 1zu2 Steckern und diese beiden Stränge dann wiederum mit Y-Kabeln auf 4 Stränge. Das Regelverhalten habe ich aber auch, wenn ich nur jeweils einen Eingang der beiden MPPT´s nutze....

Das mit den verschiedenen Leistungen ist in der Tat seltsam. Woran kann das liegen? Aktuell lasse ich den Zendure regeln, da sieht es so aus:

![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/174735302/bbe75bc9-87d0-4443-993c-b896393a8304)

Schieflage ist auch hier festzustellen. Leistung läuft aber bei Bedarf bis1200Watt (maximale Leistung HUB2000) hoch.

**Comment by Maddi1975** [comment]
Ich habe mal die Kabel der beiden MPPT´s  vertauscht. Das scheint am Kabel zu liegen... Ich werde mal schauen, wie ich das gerade gebogen bekomme.
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/174735302/27a9dcbe-7c10-4ec5-97f6-b96e2a64360e)

**Comment by Maddi1975** [comment]
spcqike, besten Dank für den Hinweis! Ich habe jetzt die Kabel an dem schwachen Strang getauscht. Nun sind alle Werte gleich und auch die Nulleinspeisung scheint zu funktionieren.
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/174735302/ba4dee04-7d49-474b-a754-ec57caf65951)
Nochmals besten Dank, da wäre ich nie drauf gekommen!!

**Reply by spcqike** [reply]
ja, der gleichmäßige und gute / feste Anschluss aller Kabel ist sehr wichtig. Da reicht manchmal schon eine nicht ganz feste Mutter, eine schlechte MC4-Stecker-Crimpung oder ein minderwertiger Zwischenstecker/Adpater :) 


Zum technischen Hintergrund, was bei dir das Problem gewesen sein kann:

die DC(-) Ausgänge am Wechselrichter sind intern alle verbunden. (lediglich die DC(+) sind einzeln oder, wie beim HM-1500, paarweise verbunden). Hat nun eine DC(-) Leitung zur Batterie einen höheren Widerstand, sucht sich der Strom halt den Weg des Geringsten. der Strom von DC3(+) kann dann auch über DC1(-) zurück fließen. Bei Solarpaneelen ist das technisch unmöglich, bei Batterien hingegen quasi immer der Fall. 
Nun messen die Wechselrichter die Eingangsleistung im DC(-) Strang eines jeden Strings. 

Dadurch passiert es, dass der Strom bspw. in DC3/4 rein kommt, durch vom MPPT umgesetzt wird aber über DC1/2 abfließt und dort als Leistung gezählt wird. Dadurch drosselt der WR die echte Leistung auf DC1/2, setzt dort weniger Strom um, er zählt ja aber noch den "Abfluss" von DC3/4 als imaginäre Produktion auf DC1/2. 

wie auch immer, unsaubere Verbindungen und unterschiedliche Leitungswiderstände (über alle Verbindungen, Leitungen zwischen WR und Batterie hinweg) führen zu solchen Schweinerein :)
